.github/workflows/lint-test.yml

```diff
@@ -6,15 +6,15 @@ on:
       - main
   pull_request:
     branches:
-      - "*"
```

Comment:
@axelboc Do you know what is the standard quote type for such files ?

I get inconsistent formatting depending on the environment

Reply:
Yeah sorry, ""depending on the environment" meant "depending on the installed VSCode extensions". I guess I have conflicting formatters installed.

Reply:
OK I found it: `"prettier.singleQuote": true` setting was missing

Reply:
Might be good to add a `.vscode` folder to enable/disable/configure formatter extensions. You've configured Prettier, but you still might have multiple formatters running on YAML files, which might cause new issues in the future.